src/org/javarosa/core/api/ILogger.java

```diff
@@ -15,38 +15,69 @@
  */
 
 /**
- * 
+ *
  */
 package org.javarosa.core.api;
 
-import org.javarosa.core.log.IFullLogSerializer;
-import org.javarosa.core.log.StreamLogSerializer;
-
 import java.io.IOException;
 import java.util.Date;
+import org.javarosa.core.log.IFullLogSerializer;
+import org.javarosa.core.log.StreamLogSerializer;
 
 /**
  * IIncidentLogger's are used for instrumenting applications to identify usage
  * patterns, usability errors, and general trajectories through applications.
- * 
- * @author Clayton Sims
- * @date Apr 10, 2009 
  *
+ * @author Clayton Sims
+ * @deprecated Use {@link org.slf4j.LoggerFactory#getLogger(Class)} instead
```

Comment:
So, I just want to maybe spread a little code-culture here. I would try as much as possible to keep the little edits like you have here in separate commits, clearly explained by their corresponding commit message. It makes reviewing and understanding the scope of commits *way* easier because as soon as you see something that doesn't match the commit message you immediately can question it. Basically, this makes review fatigue immediate and code quality usually suffers as a result. You shouldn't be afraid to do the cleanup, just organize it in incremental commits. Just in case, it's not clear what I'm talking about...

You simultaneously did the following:

- Minor code formatting (whitespace)
- Reorganized imports
- Eliminated the  `@date` javadoc tag 
- Added the `@Deprecated` annotation
- Added the `@deprecated` javadoc tag

You can still make all these changes, but by reorganizing them you can eliminate the need for questions like Dave's here by having the commit message explain exactly *why* you're doing them. It also makes it easy to see your progression by your commit log. Basically, my commit log would look like the above bullets, except I'd reword the last 3 descriptions to explain the reasoning, not the action (which you can see easily using git diff, so the commit message adds little value):

- Eliminated the  `@date` javadoc tag -> Eliminated low-value javadoc noise? (guessing at reason)
- Added the `@Deprecated` annotation -> Annotated recently retired logging api as deprecated
- Added the `@deprecated` javadoc tag -> Provided deprecation suggestions using javadoc tags

You could argue that the last two could be combined, and I agree if kept clean and explainable by:

- Annotated recently deprecated api with javadoc suggestions

Reply:
This comment is pure gold. Thanks, @batkinson!